recipe/meta.yaml

```diff
@@ -44,7 +44,7 @@ requirements:
     - libpng
     - zlib
     - libraw
-    - libwebp-base
+    - libwebp-base >=1.2
```

Comment:
This will remove the global pinning. Please revert this change.

Reply:
Thanks for the note. Maybe we have to backport the fixes on windows? @traversaro ? Or move more packages to broken?

Reply:
Interesting, I wonder why in https://github.com/conda-forge/freeimage-feedstock/runs/2142344960 the libwebp-base 1.2 was installed if there is a global pin on 1.1 . However, either we move the global pin of libwebp-base to 1.2 (as it was updated in https://github.com/conda-forge/libwebp-base-feedstock/pull/4), or we backport https://github.com/conda-forge/libwebp-base-feedstock/pull/5 and https://github.com/conda-forge/libwebp-base-feedstock/pull/6 to a v1.1 branch of `libwebp-base-feedstock`, let me know which solution you prefer and I can work on it. fyi @conda-forge/libwebp-base

Reply:
I'm on mobile right now, maybe the pin for base was recently introduced? Not sure :/ ￼I think we could move the pin, and check ABI compatibility to see if we need a migration.

@isuruf what do you think?

Reply:
yes, moving the pin is preferred. I don't think we need a migration, but abi-laboratory information is incomplete. We should ask them to update.

Reply:
I removed the requirement again since we fixed the pinnings. Thanks again!